.eslintcache

```diff
@@ -1 +0,0 @@
-[{"/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/annotator/components/toolbar.js":"1","/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/annotator/guest.js":"2","/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/annotator/toolbar.js":"3","/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/doodle/doodleCanvas.js":"4"},{"size":5021,"mtime":1614880460577,"results":"5","hashOfConfig":"6"},{"size":22213,"mtime":1614880828948,"results":"7","hashOfConfig":"6"},{"size":3684,"mtime":1614881067711,"results":"8","hashOfConfig":"6"},{"size":2861,"mtime":1614880967602,"results":"9","hashOfConfig":"10"},{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"zv3vgn",{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"itfxzx","/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/annotator/components/toolbar.js",[],"/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/annotator/guest.js",[],"/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/annotator/toolbar.js",[],"/Users/willfuchs/Documents/Cal Poly 20-21/csc 402/client/src/doodle/doodleCanvas.js",[]]
```

Comment:
I see you added .eslintcache to .gitignore, there is something you have to do to make git ignore it in the same commit tho. Fine to include it here but you might still need to remove it from what git is paying attention to in the future